bundles/automation/org.eclipse.smarthome.automation.module.core/src/main/java/org/eclipse/smarthome/automation/module/core/handler/ChannelPayloadTriggerHandler.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 1997, 2015 by ProSyst Software GmbH and others.
```

Comment:
This file is not part of this PR anymore. I now have renamed the module to `ChannelEventTriggerHandler.java` and this one has in the license information: "Copyright (c) 1997, 2015 by the respective copyright holders." like the other new files which are created.